.github/CODEOWNERS

```
@@ -46,4 +46,4 @@ tasks/update_go.py                  @DataDog/agent-shared-components
 /docker-arm64/                      @DataDog/container-integrations @DataDog/agent-devx-infra
 /docker-x64/                        @DataDog/container-integrations @DataDog/agent-devx-infra
 /dev-envs/                          @DataDog/agent-devx-loops
-/devcontainer/                      @DataDog/agent-devx-loops
```

I think instead of replacing the team we should collaborate and have the 2 teams codeowner of it.
The goal is that @DataDog/agent-devx-loops  is the principal owner of the `devcontainer` because it is in their scope. And at the same time, @DataDog/container-platform can review, share knowledge and concerns about the change.

```suggestion
/devcontainer/ @DataDog/agent-devx-loops @DataDog/container-platform
```